packages/server/graphql/private/mutations/updateEmail.ts

```diff
@@ -10,19 +10,35 @@ const updateEmail: MutationResolvers['updateEmail'] = async (_source, {oldEmail,
     throw new Error('New email is the same as the old one')
   }
 
-  const user = await getUserByEmail(oldEmail)
+  const user = await pg
+    .selectFrom('User')
+    .selectAll()
+    .where('email', '=', oldEmail)
+    .executeTakeFirst()
+
   if (!user) {
     throw new Error(`User with ${oldEmail} not found`)
   }
 
-  // RESOLUTION
+  //Use identities array for the user above
+  const identitiesArray = user.identities
```

Comment:
+1 In our code base we usually destructure fields. Also the `Array` suffix is not necessary as it's typed and `identities` is already plural. So we would move l35 up here and it would look like
```ts
const {id: userId, identities} = user
```